migrations/898-comm-cc-nonuser.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `comm_thread_cc`
+    MODIFY COLUMN `user_id` int(11) unsigned NULL,
+    ADD COLUMN `nonuser_email` varchar(255) NULL;
```

Comment:
I'm concerned the mozilla_contact email addresses will be added as a permanent cc to the thread, with no way to unsubscribe.  Is my concern unfounded?    The cc really only needs to exist for that moment - if the mozilla_contact address is removed immediately after the first message it shouldn't get any further messages.